app/controllers/admins/public_servants_controller.rb

```diff
@@ -17,13 +17,25 @@ def new
   end
 
   def create
-    @admin = Admin.new(public_servant_params.merge(password: password, password_confirmation: password))
-    if @admin.save
-      AdminMailer.send_public_servant_account(admin: @admin).deliver
-      redirect_to admins_public_servants_path, notice: t('flash.public_servant.created')
+    if params[:admin][:is_comptroller] == '1'
+      @admin = Admin.new(comptroller_params.merge(password: password, password_confirmation: password))
+      if @admin.save
+        AdminMailer.send_comptroller_account(admin: @admin).deliver
+        redirect_to admins_public_servants_path, notice: t('flash.public_servant.created')
```

Comment:
Line is too long. [90/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.